x/protorev/types/keys.go

```diff
@@ -0,0 +1,12 @@
+package types
+
+const (
+	// ModuleName defines the module name
+	ModuleName = "protorev"
+
+	// StoreKey defines the primary module store key
+	StoreKey = ModuleName
+
+	// RouterKey defines the module's message routing key
+	RouterKey = ModuleName
```

Comment:
I don't think we need this.

Reply:
Is this not needed for query/message services (which are implemented in later PRs)?